save-cloud-charts/save-cloud/templates/demo-deployment.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: demo
+  labels:
+    {{- include "common.labels" (merge (dict "service" .Values.demo) .) | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      io.kompose.service: demo
+  strategy:
+    # Because of shared volume with multi-attach problem
+    type: Recreate
```

Comment:
`demo` doesn't have persistent volume mounts that can cause problems, like for backend. So specifying strategy explicitly is redundant (default value is rolling update)

Reply:
Yes, if it's intended to be persisted between redeployments

Reply:
oh god, will do it

Reply:
You also need to link pod template from Deployment and PV, see `volumes` and `volumeMounts` in backend-deployment

Reply:
Seems that it is done, but not sure, please, double-check

Reply:
Yep, saw that, the problem is with the order of lines

Reply:
Not it seems it is done lol
Problems of copy-paste